serverless/stacks/AppStack.ts

```diff
@@ -11,6 +11,26 @@ export function AppStack({ stack }: StackContext) {
       ),
     },
   });
+  const trainspaceRunTable = new Table(stack, "trainspace-run", {
+    fields: {
+      runId: "string",
+      trainspaceId: "string",
+      userId: "string",
+      timestamp: "string",
+      resultCsvUri: "string",
+      modelPtUri: "string",
+      onnxUri: "string",
+      confusionMatrixUri: "string",
+      aucRocUri: "string"
+    },
+    primaryIndex: {
+      partitionKey: "runId",
+    },
+    globalIndexes: {
+      "TrainspaceIndex": { partitionKey: "trainspaceId", sortKey: "timestamp"},
+      "UserIndex": {partitionKey: "userId"}
+    },
+  });
   const api = new Api(stack, "Api", {
     authorizers: {
       FirebaseAuthorizer: {
```

Comment:
Add trainspaceRunTable information (name, arn, etc) to stack.addOutputs to output that in the terminal

Reply:
ok. do we really need the ARN since we are creating the table from scratch?

Reply:
Prob not actually, ig just the name and anything else that would be useful for identifying and debugging the table on AWS

Reply:
added it in @dwu359